### PR TITLE
Add command palette to start fuel core and logfile setting

### PR DESCRIPTION
**Pull request by sdankel**
Adds a command to start fuel core with a setting for a log file path. The log file includes all stdout and stderr from the fuel-core process.

Default logfile:

image

image

Custom logfile:

image

image

image

related https://github.com/FuelLabs/sway-vscode-plugin/issues/53

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,12 +63,21 @@
           "markdownDescription": "Show successfully parsed tokens by the sway-lsp server as warnings. If set to false, sway-lsp will revert to only showing warnings and errors reported by the compiler.",
           "default": false,
           "type": "boolean"
+        },
+        "sway-lsp.trace.fuel-core.logfile": {
+          "description": "Set the file path for the Fuel core log output. Disabled if set to empty string.",
+          "type": "string",
+          "default": "/tmp/fuel-core.log"
```
Comment (adlerjohn) [inline package.json:70]: Just to avoid collisions a bit more
```suggestion
          "default": "/tmp/fuel-core_sway-vscode-plugin.log"
```

**Review by adlerjohn — COMMENTED** [review]
Merge conflict. Otherwise looking nice!

File: package.json [hunk 2]
```diff
@@ -88,8 +98,15 @@
...
         "command": "sway.runScript",
         "title": "Sway: Run script"
+      },
+      {
+        "command": "sway.startFuelCore",
+        "title": "Sway: Start Fuel Core"
       }
     ],
+    "config": {
+      "logfile": "/tmp/fuel-core.log"
```
Comment (adlerjohn) [inline package.json:108]: This one too

**Comment by JoshuaBatty** [comment]
Nice work @sdankel this looks very handy. Happy approve after addressing John's comments above.

**Review by adlerjohn — APPROVED** [review]
utACK

**Pull request merged**